fastlane/lib/fastlane/actions/app_store_build_info.rb

```diff
@@ -0,0 +1,198 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      LATEST_BUILD_NUMBER = :LATEST_BUILD_NUMBER
+      LATEST_VERSION_NUMBER = :LATEST_VERSION_NUMBER
+    end
+
+    class AppStoreBuildInfoAction < Action
+      def self.run(params)
+        require 'spaceship'
+
+        build_nr = get_build_number(params)
+
+        # Convert build_nr to int (for legacy use) if no "." in string
+        if build_nr.kind_of?(String) && !build_nr.include?(".")
```

Comment:
@RishabhTayal `build_nr` is a hash here so this logic should probably get reworked to lookup from the hash properly

Reply:
looking up from the hash now.